staging/src/k8s.io/apiserver/plugin/pkg/audit/webhook/webhook.go

```diff
@@ -87,8 +87,9 @@ var (
 	//
 	// Can we make these passable to NewGenericWebhook?
 	groupFactoryRegistry = make(announced.APIGroupFactoryRegistry)
-	groupVersions        = []schema.GroupVersion{auditv1alpha1.SchemeGroupVersion}
-	registry             = registered.NewOrDie("")
+	// TODO(audit): figure out a general way to let the client choose their preferred version
+	groupVersions = []schema.GroupVersion{auditv1beta1.SchemeGroupVersion}
```

Comment:
> I can't see your changes in the PR actually. It's in progress, I guess?

Because I removed the change a moment ago.
https://github.com/CaoShuFeng/kubernetes/pull/1/files#r133389297